README.md

```diff
@@ -191,13 +191,20 @@ The rest of the publications describes different component of **Crocoddyl**:
 You have a question or an issue? You may either directly open a [new issue](https://github.com/loco-3d/crocoddyl/issues) or use the mailing list <crocoddyl@laas.fr>.
 
 
+## Executive Committee
```

Comment:
Steering Committee sound better

Reply:
Steering is more connected with making higher decision.
Executive is more connected with executing those decisions.

Both are OK, I would say, but they have slightly different meanings.

Reply:
It sounds like Steering Committee is ok for both of you. I'll change it to that.